core/src/test/java/org/jclouds/proxy/ProxyForURITest.java

```diff
@@ -41,12 +42,14 @@
 
    private class MyProxyConfig implements ProxyConfig {
       private boolean useSystem;
+      private boolean canUseProxyFromJvm;
       private Type type;
       private Optional<HostAndPort> proxy;
       private Optional<Credentials> credentials;
 
-      MyProxyConfig(boolean useSystem, Type type, Optional<HostAndPort> proxy, Optional<Credentials> credentials) {
+      MyProxyConfig(boolean useSystem, boolean canUseProxyFromJvm, Type type, Optional<HostAndPort> proxy, Optional<Credentials> credentials) {
```

Comment:
Hmmm, this only enables the jvm proxy as a *fallback* if no jclouds proxy properties have been configured (which will be the most common case in jclouds applications given the current logic). I'd say that perhaps `enableJvmProxy` could be more clear, but I'm not the language expert here :)
Not a big deal for me in any case.